webapp/components/admin_console/admin_navbar_dropdown.jsx

```diff
@@ -45,6 +49,26 @@ export default class AdminNavbarDropdown extends React.Component {
         TeamStore.removeChangeListener(this.onTeamChange);
     }
 
+    componentDidUpdate() {
+        if (document.body.classList.contains('app__body') && this.state.showAboutModal === false) {
+            // Timeout needed to ensure modal closing animation can fully complete
+            setTimeout(() => {
+                $('body').removeClass('app__body');
+            }, 300);
+        }
+    }
```

Comment:
If you'd like to preserve the animation, instead of doing it this way can you convert the modal to close [like this one](https://github.com/mattermost/platform/blob/master/webapp/components/edit_channel_header_modal.jsx)? You can then use `ToggleModalButton` [similar to here](https://github.com/mattermost/platform/blob/master/webapp/components/channel_header.jsx#L372) which should preserve the animation

Also just a heads-up [we're moving away from using jQuery](https://docs.mattermost.com/developer/core-developer-handbook.html#technical-direction-resolutions)